### DISCUSSION TITLE
Show object expandable instead of static object

### DISCUSSION THREAD

**Discussion by jonatandorozco** [permafrost-dev/node-ray#14] [Ideas]
Could be awesome if we can see an object and expand its properties like we can do with php/laravel version

**Comment by patinthehat** [comment]
This is something I am looking into, stay tuned.

**Reply by jonatandorozco** [reply]
I was checking out the way the Laravel implementation do this

It generates an script. Looks kind of predictable

Let me know if I can help with something about this

**Reply by patinthehat** [reply]
@jonorozcoc This feature will require modification of the [permafrost-dev/pretty-format](https://github.com/permafrost-dev/pretty-format) package - I'd be happy to accept a PR.
